src/displayapp/screens/Twos.cpp

```diff
@@ -241,8 +241,8 @@ void Twos::updateGridDisplay() {
     const unsigned int row = i / nCols;
     const unsigned int col = i % nCols;
     if (grid[row][col].value > 0) {
-      char buffer[7];
-      sprintf(buffer, "%d", grid[row][col].value);
+      char buffer[11];
```

Comment:
nitpick: could we leave this buffer to be of size 7. I don't want to increase the buffer size for the game as the buffers are copied into the `lv_table`.

Keeping the  size of 7 results in 6 usable bytes. Then the biggest possible value in the game would be `2**19` -> `524288`.

I doubt that anyone would reach this insane number while playing the game

Even buffer size of 6 (with 5 usable bytes) would be enough to hold `2**16 = 65536`. But I think keeping the size that works is less intrusive than saving a few bytes

Reply:
Because there are 16 squares and the largest number you can get is 4, it is not possible to have a tile with a value above 2^17 (131072). So 7 bytes is guaranteed to be sound